### PR TITLE
Add Font and Mesh icons that aren't grayed out

### PR DESCRIPTION
**Pull request by MewPurPur**
Font is an abstract class, so it uses a gray icon now. But some places used the class icon and aren't supposed to look disabled. Fixes this:

image

### CODE REVIEW

**Comment by YuriSizov** [comment]
That's exactly what I was referring to in https://github.com/godotengine/godot/pull/80184#issuecomment-1664315117 😛 

> These icons may be used for things other than node types in the editor. And in those cases it is very likely important that icons have their assigned color palette

**Review by YuriSizov — DISMISSED** [review]

**Comment by MewPurPur** [comment]
Ooh, I see now. I got confused with the English, I thought you meant that in the future, they might be used for something else (as in, a feature or sth). But I think it's fine to split off into multiple icons for the different contexts.

Alright, wait a sec, I'll check if there might be other icons with this issue.

**Comment by MewPurPur** [comment]
Mesh also popped up in a few other places. It used to be yellow, now I made the new variant white (afaik this seems fine, given the context, but I'm not heavy on 3D scene importing)

IMO it's best to have "de facto" class icons that are shown everywhere the same, even if it is slightly more taxing. I think otherwise we'd have to think about all the places modulation is needed or not.

File: editor/plugins/theme_editor_plugin.cpp [hunk 1]
```diff
@@ -2556,7 +2556,7 @@ void ThemeTypeEditor::_update_type_items() {
 			HBoxContainer *item_control = _create_property_control(Theme::DATA_TYPE_FONT, E.key, E.value);
 			EditorResourcePicker *item_editor = memnew(EditorResourcePicker);
 			item_editor->set_h_size_flags(SIZE_EXPAND_FILL);
-			item_editor->set_base_type("Font");
+			item_editor->set_base_type("FontItem");
```
Comment (AThousandShips) [inline editor/plugins/theme_editor_plugin.cpp:2559]: Is this correct? This should be a type not the icon no?

Comment (AThousandShips) [reply editor/plugins/theme_editor_plugin.cpp:2559]: Looks like a search and replace error, I know all about them myself!

Comment (AThousandShips) [reply editor/plugins/theme_editor_plugin.cpp:2559]:
```suggestion
			item_editor->set_base_type("Font");
```

Comment (MewPurPur) [reply editor/plugins/theme_editor_plugin.cpp:2559]: > Looks like a search and replace error, I know all about them myself!

No, I was just reckless.

**Review by YuriSizov — APPROVED** [review]

**Pull request merged**